README.md

```diff
@@ -32,22 +38,34 @@ config :bugsnex, :use_logger, true
 ## Usage
 
 ### Sending bug reports
+
 Once configured, use `Bugsnex.notice(exception)` or `Bugsnex.notice(exception,stacktrace)` to send errors to Bugsnag.
 
 If `use_logger` is set to `true`, an [error logger](http://erlang.org/doc/man/error_logger.html) event handler is added
 and [SASL](http://erlang.org/doc/apps/sasl/error_logging.html) compliant errors are sent to Bugsnag.
 
 You can also manually wrap code in a `Bugsnex.handle_error` block. Errors in this block will then be sent to Bugsnag and reraised. Example:
 
-
 ```elixir
 Bugsnex.handle_errors %{some: "metadata"} do
   somthing_that_could_raise_and_error()
 end
 ```
 
+Bugsnex also provides a Plug called `Bugsnex.Plug` that you could add to
```

Comment:
I think the plug is important enough to warrant its own `### Automatic reports via Bugsnex.Plug` or something more beautiful than that heading. It's a minor thing, I can also add it manually :)